api_app/models/domain/airlock_request.py

```diff
@@ -81,16 +81,17 @@ class AirlockRequest(AzureTREModel):
     """
     id: str = Field(title="Id", description="GUID identifying the resource")
     resourceVersion: int = 0
-    user: dict = {}
+    createdBy: dict = {}
+    createdWhen: float = Field(None, title="Creation time of the request")
+    updatedBy: dict = {}
     updatedWhen: float = 0
     history: List[AirlockRequestHistoryItem] = []
     workspaceId: str = Field("", title="Workspace ID", description="Service target Workspace id")
-    requestType: AirlockRequestType = Field("", title="Airlock request type")
+    type: AirlockRequestType = Field("", title="Airlock request type")
```

Comment:
Does this affect the payload shape in the API? I.e. do callers now need to pass `type` instead of `requestType`?

Reply:
It does indeed - it changes from:
```
{
    "requestType": "import",
    "requestTitle": "a request title",
    "businessJustification": "some business justification"
}
```

to 

```
{
    "type": "import",
    "title": "a request title",
    "businessJustification": "some business justification"
}
```

Reply:
Ok, I think we should call this out in breaking changes in the CHANGELOG.

Also, can you update the CLI as this change will break it

Reply:
Yes good shout, will do :)

Reply:
And already updated the CLI ;)

Reply:
Changelog updated